engine/moduleManager/ModuleManager.hpp

```diff
@@ -10,7 +10,7 @@
 #include <unordered_map>
 #include <stdexcept>
 
-#include "moduleManager/ModuleWrapper.hpp"
```

Comment:
Waarom hier moduleManager weghalen? technisch gezien niet nodig gezien ze in dezelfde directory zitten, maar heb ik overal gedaan voor de duidelijkheid

Reply:
Ja dat is prima, dan houden we dat zo